.github/workflows/acquia_cms_ci.workflow.yml

```diff
@@ -98,7 +98,7 @@ jobs:
     strategy:
       matrix:
         orca-job:
-          - ISOLATED_TEST_ON_CURRENT_DEV
+          - ISOLATED_TEST_ON_NEXT_MINOR
```

Comment:
@deepakmishra2 Please revert this (and below as well) as we should test on Drupal core 9.5 seperately.